README.md

```diff
@@ -10,6 +10,8 @@ Currently supported React Native version: `^0.53.3`
 $ react-native init HelloWorld --template reason
 ```
 
+You can run the [playground](./example) with Expo: https://expo.io/@grabbou/bs-react-native
```

Comment:
Does this still work on iOS? I vaguely recall Apple banning the ability to run apps that you don't own in Expo. 

If that's the case it should probably be documented that this only works on android

Reply:
Agree, I put information here